bin/informational/overdrive-raw-bibliographic

```diff
@@ -7,17 +7,24 @@ bin_dir = os.path.split(__file__)[0]
 package_dir = os.path.join(bin_dir, "..", "..")
 sys.path.append(os.path.abspath(package_dir))
 
+from core.model import (
+    Collection,
+    ExternalIntegration,
+)
```

Comment:
This change is unrelated; I needed to run this script and it hadn't been updated to be collection-aware like bibliotheca-raw-bibliographic had.